src/containers/DefaultLayout/DefaultFooter.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['prop-types' should be listed in the project's dependencies. Run 'npm i -S prop-types' to add it](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)